models/aave/optimism/aave_optimism_sources.yml

```diff
@@ -30,12 +31,110 @@ sources:
           description: "Liqudity index value of the reserve"
         - name: liqudityRate
           description: "Liqudity rate value of the reserve"
-        - name: reserve
+        - &reserve
+          name: reserve
           description: "Aave reserve contract address"
         - name: stableBorrowRate
           description: "Stable borrow rate value of the reserve"  
         - name: variableBorrowIndex
           description: "Variable borrow index value of the reserve"
         - name: variableBorrowRate
-          description: "Variable borrow rate value of the reserve"   
+          description: "Variable borrow rate value of the reserve"     
+
+      - name: Pool_evt_Supply
+        loaded_at_field: evt_block_time

```

Comment:
add freshness for this and the following source tables?

Reply:
Added